Schema-validation: wrong column type encountered in column [DIRECT_DEPENDENCIES] in table [`PROJECT`];

### Current Behavior

After deploying the helm chart with very basic parameters, we are going to have just 1 component failing, which is: "hyades-mirror-service". With the error below in the logs. I believe it's a migration issue.

```
		at io.quarkus.runtime.Quarkus.run(Quarkus.java:124)
	at io.quarkus.runner.GeneratedMain.main(Unknown Source)
Caused by: jakarta.persistence.PersistenceException: [PersistenceUnit: <default>] Unable to build Hibernate SessionFactory
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.persistenceException(FastBootEntityManagerFactoryBuilder.java:129)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:89)
	at io.quarkus.hibernate.orm.runtime.FastBootHibernatePersistenceProvider.createEntityManagerFactory(FastBootHibernatePersistenceProvider.java:73)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$LazyPersistenceUnit.get(JPAConfig.java:154)
	at io.quarkus.hibernate.orm.runtime.JPAConfig$1.run(JPAConfig.java:61)
	at java.base@21.0.4/java.lang.Thread.runWith(Thread.java:1596)
	at java.base@21.0.4/java.lang.Thread.run(Thread.java:1583)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:896)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:872)
Caused by: org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: wrong column type encountered in column [DIRECT_DEPENDENCIES] in table [`PROJECT`]; found [jsonb (Types#OTHER)], but expecting ["varchar" (Types#VARCHAR)]
	at org.hibernate.tool.schema.internal.AbstractSchemaValidator.validateColumnType(AbstractSchemaValidator.java:169)
	at org.hibernate.tool.schema.internal.AbstractSchemaValidator.validateTable(AbstractSchemaValidator.java:156)
	at org.hibernate.tool.schema.internal.GroupedSchemaValidatorImpl.validateTables(GroupedSchemaValidatorImpl.java:46)
	at org.hibernate.tool.schema.internal.AbstractSchemaValidator.performValidation(AbstractSchemaValidator.java:99)
	at org.hibernate.tool.schema.internal.AbstractSchemaValidator.doValidation(AbstractSchemaValidator.java:77)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:289)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.lambda$process$5(SchemaManagementToolCoordinator.java:144)
	at java.base@21.0.4/java.util.HashMap.forEach(HashMap.java:1429)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:141)
	at io.quarkus.hibernate.orm.runtime.observers.SessionFactoryObserverForSchemaExport.sessionFactoryCreated(SessionFactoryObserverForSchemaExport.java:21)
	at org.hibernate.internal.SessionFactoryObserverChain.sessionFactoryCreated(SessionFactoryObserverChain.java:35)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:323)
	at io.quarkus.hibernate.orm.runtime.boot.FastBootEntityManagerFactoryBuilder.build(FastBootEntityManagerFactoryBuilder.java:87)
	... 9 more
2024-09-18T20:06:58.071439906Z
```


### Steps to Reproduce

1. Use the following Terraform code:
```
resource "helm_release" "default" {
  name       = "app"
  chart      = "hyades"
  repository = "https://dependencytrack.github.io/helm-charts"

  namespace        = var.namespace
  create_namespace = true

  set {
    name  = "common.database.jdbcUrl"
    value = var.jdbc_url
  }

  set {
    name  = "common.database.username"
    value = var.jdbc_username
  }

  set {
    name  = "common.database.password"
    value = var.jdbc_password
  }

  set {
    name  = "common.kafka.bootstrapServers"
    value = var.queue_brokers
  }

  timeout = 150 # means 2,5 minutes
}
```

2. And apply, that's it. You are going to see the helm installed and the error throwing.

### Expected Behavior

I was expecting to have everything working fine. Since Kafka and PostgreSQL are fresh new deployments.

### Hyades Version

Chart: hyades-0.7.0
Version: 0.7.0

### Repository Type

Hyades services

### Browser

Google Chrome

### Checklist

- [X] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/hyades/blob/master/CONTRIBUTING.md#filing-issues)
- [X] I have checked the [existing issues](https://github.com/DependencyTrack/hyades/issues) for whether this defect was already reported

--- a/.github/workflows/build-native-mirror-service.yml
+++ b/.github/workflows/build-native-mirror-service.yml
@@ -26,6 +26,7 @@ on:
     - ".github/workflows/build-native-mirror-service.yml"
     - "commons/**"
     - "commons-kstreams/**"
+    - "commons-persistence/**"
     - "mirror-service/**"
     - "pom.xml"
     - "proto/**"

